### DISCUSSION TITLE
Woocommerce checkout process

### DISCUSSION THREAD

**Discussion by RobertCrx** [bootscore/bootscore#576] [Q&A] [answered]
I have an issues with the checkout field ``` email ```. If someone is logged in and have an account linked with his xyz@yahoo.com , I cannot use that email address again to receive another order.. It says 'This email address is already in use. Please use another email address.' which is stupid because it is my main email address.
I have to let users use any email addresses.. I tried different woocommerce filters to remove errors , but they have no effect. Any idea ?

**Comment by crftwrk** [comment, marked as answer]
Hmm, we did no changes there. Think the [WC discussions](https://github.com/woocommerce/woocommerce/discussions) are a better place to get an answer.

**Comment by justinkruit** [comment]
This isn't something bootScore touches on. But it seems like this is default woocommerce behavior. Is there maybe an account on your woocommerce webshop which uses that same email? It may be that woocommerce doesn't allow to place an order using that mail address, because you're not signed in and it detects there is already an user with that mail address.

**Comment by RobertCrx** [comment]
I had server cache sorry , it was a filter that verify duplicates.
